libs/blocks/global-navigation/utilities/keyboard/index.js

```diff
@@ -51,12 +51,14 @@ class KeyboardNavigation {
           break;
         }
         case 'Enter': {
+          if (e.target.classList.contains(selectors.searchField.replace('.', ''))) return;
```

Comment:
Would `closest` not be an option here to avoid the `replace`, something like `if (e.target.closest(selectors.searchField))`.